<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>class_diagram</class>
 <widget class="QMainWindow" name="class_diagram">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1505</width>
    <height>815</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>class_diagram</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>20</y>
      <width>1750</width>
      <height>1000</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>1750</width>
      <height>1000</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>260</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>131</width>
      <height>211</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLineEdit" name="lineEdit_2">
       <property name="text">
        <string>D:\VUT\6_Semester\ICP\examples\sequence.json</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="NewClass">
       <property name="text">
        <string> Insert Sequence 
  Diagram</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit">
       <property name="text">
        <string>D:\VUT\6_Semester\ICP\examples\class.json</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_2">
       <property name="text">
        <string>Insert Class 
 Diagram</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>330</y>
      <width>131</width>
      <height>221</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Add new Class</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="AddName">
       <property name="text">
        <string>name</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="AddClass">
       <property name="text">
        <string>Add</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Remove Class</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="DelName">
       <property name="text">
        <string>name</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="DelClass">
       <property name="text">
        <string>Delete</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="Save">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>610</y>
      <width>80</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Save</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="imageName">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>580</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1505</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
